doc/whats_new.rst

```diff
@@ -137,6 +137,10 @@ Enhancements
    - The :func: `ignore_warnings` now accept a category argument to ignore only
      the warnings of a specified type. By `Thierry Guillemot`_.
 
+   - Added Labels flag to `metrics.log_loss` to correct metrics when only one class is present
```

Comment:
lowercase labels, and surround it with double backticks (see line 129 for how to format parameters)